flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/util/MergeUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.contrib.streaming.state.util;
+
+import org.apache.flink.annotation.VisibleForTesting;
+
+import java.util.List;
+
+/**
+ * Utils to simulate StringAppendTestOperator's merge operations in RocksDB.
+ */
+public class MergeUtils {
+	@VisibleForTesting
+	protected static final byte DELIMITER = ',';
```

Comment:
It would be great if we could initialize this from the RocksDB API somehow. But not sure if this is possible.

Reply:
RocksDB doesn't expose that

Reply:
Ok, that is unfortunate.